playbooks/roles/cluster/templates/workflow_survey.json

```diff
@@ -0,0 +1,51 @@
+{
+    "description": "",
+    "name": "",
+    "spec": [
+      {
+        "question_description": "Desired name of Openshift cluster",
+        "min": 0,
+        "default": "",
+        "max": 16,
+        "required": true,
+        "choices": "",
+        "variable": "oo_clusterid",
+        "question_name": "Cluster Name",
+        "type": "text"
+      },
+      {
+        "question_description": "AWS account name",
+        "min": 0,
+        "default": "aws.eng",
+        "max": 100,
+        "required": false,
+        "choices": "",
```

Comment:
Should be able to set dynamic choices in the survey by listing existing AWS creds. We do it already for the integreatly workflow survey. 

See examples:
https://github.com/integr8ly/ansible-tower-configuration/blob/master/playbooks/roles/integreatly/tasks/workflow.yml#L39-L54
https://github.com/integr8ly/ansible-tower-configuration/blob/master/playbooks/roles/integreatly/templates/workflow_survey.json.j2#L105